core/templates/dev/head/components/VersionDiffVisualizationDirective.js

```diff
@@ -258,7 +258,7 @@ oppia.directive('versionDiffVisualization', [function() {
 
               if (newState) {
                 $http.post(STATE_YAML_URL, {
-                  state_dict: newState,
+                  state_dict: newState.toBackendDict(),
```

Comment:
The change at line 453 of explorationSaveService changed newState from a dict to an object